cEP-0017.md

```diff
@@ -0,0 +1,89 @@
+cib Design and Improvements
+========================
+
+|Metadata      |                                                |
+|--------------|------------------------------------------------|
+| cEP          | 17                                             |
+| Version      | 1.0                                            |
+| Title        | cib Design and Improvements                     |
+| Authors      | Saksham Bansal <sakshambansal1999@yahoo.com>   |
+| Status       | Implementation due                             |
+| Type         | Process                                        |
+
+# Abstract
+
+This cEP describes the design of the cib tool which will allow it to install
+individual bears and their dependencies easily.
+
+The cEP also suggests changes to  Requirement classes and using conda packages
```

Comment:
repeated spaces.
Also `Requirement` should be in backticks.  Lots of backticks needed throughout this cEP